simulate_scanning_strategy.py

```diff
@@ -205,26 +214,42 @@ def main():
         # fractions of an arcmin over a time span of one hour)
         time0 = times[0]
         icrs_pos = get_ecliptic_vec(get_body_barycentric(params.planet_name, time0))
-        sat_pos = get_ecliptic_vec(get_body_barycentric("earth", time0))
+        earth_pos = get_ecliptic_vec(get_body_barycentric("earth", time0))
 
         # Move the spacecraft to L2
-        sat_pos = sat_pos * (
-            1.0 + params.earth_L2_distance_km / norm(sat_pos).to("km").value
+        L2_pos = earth_pos * (
+            1.0 + params.earth_L2_distance_km / norm(earth_pos).to("km").value
+        )
+
+        # Creating a circular orbit
+        scann = sim.parameters["scanning_strategy"]
+        R = conversion(scann["radius"], "au")
+        phi_t = scann["angular_velocity_rad"] * t
```

Comment:
This parameter should be moved in the `Parameter` class as well. A clearer name would probably be `L2_orbital_velocity_rad_s`.